sdk/core/azure-core/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Added `Azure::PagedResponse<T>`.
 
+### Breaking Changes
+
+- Removed `Context::GetApplicationContext()` in favor of a new static data member `Context::ApplicationContext`.
```

Comment:
@antkmsft can you help me understand what is the customer facing benefit of this change?

Reply:
Data member is simpler than method call.